crates/bitwarden-json/Cargo.toml

```diff
@@ -17,7 +17,7 @@ rust-version = "1.57"
 internal = ["bitwarden/internal"] # Internal testing methods
 
 [dependencies]
-schemars = ">=0.8.12, <0.9"
+schemars = { version = ">=0.8.12, <0.9", features = ["uuid1", "chrono"] }
```

Comment:
Do we need `uuid` here?

Reply:
I've added it because the `SessionLogin` command contains an UUID internally, but that's defined in the `bitwarden` crate, so it's not needed here indeed.

Technically we're inheriting both features from the `bitwarden` crate, so we could remove them both to avoid duplication anyway.

Reply:
Hmm, if the json crate itself needs uuid1 we should keep it.